src/main/java/org/graylog/integrations/aws/resources/requests/CreateLogSubscriptionPolicyRequest.java

```diff
@@ -0,0 +1,45 @@
+package org.graylog.integrations.aws.resources.requests;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import org.graylog.autovalue.WithBeanGetter;
+
+@JsonAutoDetect
+@AutoValue
+@WithBeanGetter
+public abstract class CreateLogSubscriptionPolicyRequest implements AWSRequest {
+
+    private static final String STREAM_NAME = "stream_name";
+    private static final String STREAM_ARN = "stream_arn";
+    private static final String ROLE_NAME = "role_name";
```

Comment:
this is more of a preference, but `ROLE_NAME` could be listed first above `STREAM_NAME`, to keep consistent with the rest of the class

Reply:
Great catch! Agreed.